cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp

```diff
@@ -32,19 +32,32 @@
     <p><spring:message code="screen.cookies.disabled.message" /></p>
 </div>
 
+
 <c:if test="${not empty registeredService}">
     <c:set var="registeredServiceLogo" value="images/webapp.png"/>
-    <c:if test="${not empty registeredService.logo}">
-        <c:set var="registeredServiceLogo" value="${registeredService.logo}"/>
-    </c:if>
+    <c:set var="registeredServiceName" value="${registeredService.name}"/>
+    <c:set var="registeredServiceDescription" value="${registeredService.description}"/>
+
+    <c:choose>
```

Comment:
This is sort of consistent with how we do with CAS service's logo, and etc. These are elements that belong in the UI. They also provide flexibility for the adopter and allow him/her to decide how the UI elements should be organized if one was not available, but the other was,  etc.